ChromeExtension/options.html

```diff
@@ -2,6 +2,7 @@
 <html lang="fr" ng-app="radio-play">
```

Comment:
Test de commentaire ailleurs que sur la modif
```suggestion
<html lang="fr" ng-app="radio-play">
```